### ISSUE TITLE
Exponent notation is used even on numbers inside the excluded exponent range

### ISSUE THREAD

**Issue by agostonbarna** [mcastorina/graph-cli#52]
Hi,
I ran into an issue that the exponent notation is used in some cases with +1e4 or larger numbers that are inside the default `--exponent-range=-3:9`. It shouldn't be neccessary but I also tried to adjust the exponent range to a larger/smaller number above +1e4, but it had no effect on the issue.

Let me provide some minimal test cases using the following command:

```sh
graph input.csv -F %Y-%m-%d --title ' ' --xlabel 'x' --ylabel 'y' --xscale 1 -o out.png
```

## Test case 1 => fail

Input:
```csv
date,a,b
2021-09-29,10000,10001
2021-09-30,10000,10001
2021-10-01,10000,10001
```
Output:
![graph](https://user-images.githubusercontent.com/5561396/135719534-5a174949-eb5b-433d-8f14-780cfa9c1428.png)

## Test case 2 => fail

Input:
```csv
date,a,b
2021-09-29,100000,100001
2021-09-30,100000,100001
2021-10-01,100000,100001
```
Output:
![graph](https://user-images.githubusercontent.com/5561396/135719578-ed33154f-2764-4e6c-b009-19a93a063c20.png)

## Test case 3 => fail

Input:
```csv
date,a,b
date,a,b
2021-09-29,100000,100010
2021-09-30,100000,100010
2021-10-01,100000,100010
```
Output:
![graph](https://user-images.githubusercontent.com/5561396/135719604-c11f2073-534a-4040-a933-498de2018391.png)

## Test case 4 => pass

Input:
```csv
date,a,b
2021-09-29,1000,1001
2021-09-30,1000,1001
2021-10-01,1000,1001
```
Output:
![graph](https://user-images.githubusercontent.com/5561396/135719789-adc912ef-70c3-4db6-aac1-f885c488cf90.png)

## Test case 5 => pass

Input:
```csv
date,a,b
2021-09-29,10000,10010
2021-09-30,10000,10010
2021-10-01,10000,10010
```
Output:
![graph](https://user-images.githubusercontent.com/5561396/135719698-b3ece0d9-cc8f-492b-9df0-58052fe8c10c.png)

## Test case 6 => pass

Input:
```csv
date,a,b
2021-09-29,10000,100100
2021-09-30,10000,100100
2021-10-01,10000,100100
```
Output:
![graph](https://user-images.githubusercontent.com/5561396/135719766-6f574b00-305c-49b5-ac8b-b286ad451856.png)

---

I didn't see an option for this, but is it possible to entirely disable the scientific notation? Because actually that is my end goal.

Thanks!

## Additional info

I have the latest graph-cli version installed with pip: v0.1.13

**Comment by mcastorina** [comment]
Thanks for reporting! It turns out I'm not formatting the Y-axis if it's a time-series graph.

I'll push v0.1.14 soon with the fix.

**Comment by mcastorina** [comment]
Fixed in 2fe1273e7a1dac95a12bcc9a8cf800aaba9c1749

`v0.1.14` is on PyPI with the changes. I didn't test all your cases, but I believe it should work. Feel free to re-open this issue if I'm mistaken.

**Comment by agostonbarna** [comment]
Wow, thanks for the quick fix @mcastorina !
Looking good, all my tests passed now :+1:
